src/org/omegat/gui/glossary/GlossaryTextArea.java

```diff
@@ -451,4 +446,29 @@ public void actionPerformed(ActionEvent e) {
                 .setPreference(Preferences.GLOSSARY_SORT_BY_LENGTH, sortOrderLocLength.isSelected()));
         menu.add(sortOrderLocLength);
     }
+
+    private static JMenuItem createMenuItem(String titleKey, String command, Runnable action) {
+        return createMenuItem(titleKey, command, ev -> {
+            Log.logInfoRB("LOG_MENU_CLICK", ev.getActionCommand());
+            action.run();
+        });
+    }
+
+    private static JMenuItem createMenuItem(String titleKey, String command, ActionListener actionListener) {
+        JMenuItem result = new JMenuItem();
+        setLocalizedText(result, OStrings.getString(titleKey));
+        result.setActionCommand(command);
```

Comment:
I think also action command is not necessary. In the past it was used when same action listener was shared by multiple menus, which is the case in MainWindowMenu actually.
But with a distinct lambda for each menu, does not seem necessary